std/digest/digest.d

```diff
@@ -579,6 +579,15 @@ interface Digest
         @trusted nothrow void put(scope const(ubyte)[] data...);
 
         /**
+         * This is a conveniance function to pass some strings with char
+         * as element encoding type.
+         */
+        @trusted nothrow final void putText(scope const(char)[] data...)
```

Comment:
Why not do the same thing here? Just have a put overload for non-ubyte, then wrap the ubyte version.

For example, `put(byte[])` should also work.